tests/src/podman-detection-smoke.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { platform } from 'os';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let navBar: NavigationBar;
+let dashboardPage: DashboardPage;
+let settingsBar: SettingsBar;
+const os = platform();
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoName('podman-detection');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navBar = new NavigationBar(page); // always present on the left side of the page
+  dashboardPage = await navBar.openDashboard();
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+});
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+describe('Podman Detection verification', async () => {
+  // TODO: add skipif in case where we expect podman to be installed, not running
+  describe.runIf(os === 'linux')('On Linux', async () => {
+    test('Podman is installed and running on Dashboard', async () => {
+      const podmanProvider = dashboardPage.getPodmanStatusLocator();
+      await playExpect(podmanProvider).toBeVisible();
+
+      // TODO: missing ARIA functionality
+      const actualState = podmanProvider.getByLabel('Actual State');
+      await playExpect(actualState).toBeVisible();
+      await playExpect(actualState).toContainText('running', { ignoreCase: true });
+      const providerVersion = podmanProvider.getByLabel('Provider Version');
+      await playExpect(providerVersion).toBeVisible();
+      await playExpect(providerVersion).toContainText(/v\d.\d.\d/, { ignoreCase: true });
+    });
+
+    test('Podman provider is running in Resources', async () => {
+      settingsBar = await navBar.openSettings();
+      const resourcesPage = await settingsBar.openTabPage(ResourcesPage);
+      const podmanBox = resourcesPage.featuredProviderResources.getByRole('region', {
+        name: 'podman',
+      });
+      await playExpect(podmanBox).toBeVisible();
+
+      const setupButton = podmanBox.getByRole('button', { name: 'Setup Podman' });
+      await playExpect(setupButton).toBeVisible();
+      const connectionStatusLabel = podmanBox.getByLabel('connection-status-label');
+      await playExpect(connectionStatusLabel).toBeVisible();
+      await playExpect(connectionStatusLabel).toContainText('RUNNING');
+      const podmanTypeLabel = podmanBox.getByLabel('Podman type');
+      await playExpect(podmanTypeLabel).toBeVisible();
+      await playExpect(podmanTypeLabel).toContainText('Podman endpoint');
+      const podmanEndpointLabel = podmanBox.getByLabel('Podman endpoint');
+      await playExpect(podmanEndpointLabel).toBeVisible();
+      await playExpect(podmanEndpointLabel).toContainText('podman.sock');
+    });
+  });
+  describe.runIf(os === 'win32')('On Windows', async () => {
+    test('Podman is installed and machine started', async () => {
+      // TODO add windows and mac specific test
```

Comment:
Unimplemented tests shouldn't be merged into master, probably best to set the skip parameters to this describe block or remove it completely for now.